<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>575</width>
    <height>869</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>575</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Submitter Main</string>
  </property>
  <widget class="QWidget" name="central_widget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="leftMargin">
     <number>15</number>
    </property>
    <property name="topMargin">
     <number>15</number>
    </property>
    <property name="rightMargin">
     <number>15</number>
    </property>
    <property name="bottomMargin">
     <number>15</number>
    </property>
    <item>
     <widget class="QGroupBox" name="basic_info_grp">
      <property name="title">
       <string>Basic Info</string>
      </property>
      <property name="flat">
       <bool>true</bool>
      </property>
      <property name="checkable">
       <bool>false</bool>
      </property>
      <layout class="QFormLayout" name="formLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="job_name_label">
         <property name="text">
          <string>Job Name</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="job_type_label">
         <property name="text">
          <string>Job Type</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="job_name_line">
         <property name="placeholderText">
          <string>Job Name</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QComboBox" name="job_type_combo"/>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="reqs_label">
         <property name="text">
          <string>Requirements</string>
         </property>
        </widget>
       </item>
       <item row="5" column="1">
        <layout class="QGridLayout" name="reqs_grid"/>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="maya_render_grp">
      <property name="title">
       <string>Maya Render</string>
      </property>
      <property name="flat">
       <bool>true</bool>
      </property>
      <property name="checkable">
       <bool>false</bool>
      </property>
      <layout class="QFormLayout" name="formLayout_5">
       <item row="0" column="0">
        <widget class="QLabel" name="maya_render_scene_label">
         <property name="text">
          <string>Scene</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <layout class="QHBoxLayout" name="maya_render_scene_grp">
         <item>
          <widget class="QLineEdit" name="maya_render_scene_line">
           <property name="placeholderText">
            <string>Maya Scene Path</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="maya_render_scene_btn">
           <property name="text">
            <string>Browse</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="maya_render_frame_label">
         <property name="text">
          <string>Frame Range</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <layout class="QHBoxLayout" name="maya_render_frame_grp">
         <item>
          <widget class="QSpinBox" name="maya_render_start_frame_spin">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
             <horstretch>1</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimum">
            <number>1</number>
           </property>
           <property name="maximum">
            <number>99999</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="maya_render_dash1">
           <property name="text">
            <string>-</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="maya_render_end_frame_spin">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
             <horstretch>1</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimum">
            <number>1</number>
           </property>
           <property name="maximum">
            <number>99999</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="maya_render_dash2">
           <property name="text">
            <string>:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="maya_render_by_frame_spin">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
             <horstretch>1</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimum">
            <number>1</number>
           </property>
           <property name="maximum">
            <number>99999</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="maya_render_batch_info_label">
           <property name="minimumSize">
            <size>
             <width>80</width>
             <height>0</height>
            </size>
           </property>
           <property name="text">
            <string>1 Frames</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="maya_render_rl_label">
         <property name="text">
          <string>Render Layers</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QTreeWidget" name="maya_render_rl_tree">
         <property name="alternatingRowColors">
          <bool>true</bool>
         </property>
         <column>
          <property name="text">
           <string>Render Layers</string>
          </property>
         </column>
        </widget>
       </item>
       <item row="5" column="1">
        <layout class="QHBoxLayout" name="add_rl_grp">
         <item>
          <spacer name="add_rl_spacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="add_rl_btn">
           <property name="minimumSize">
            <size>
             <width>100</width>
             <height>0</height>
            </size>
           </property>
           <property name="text">
            <string>Add Layer</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="remove_rl_btn">
           <property name="minimumSize">
            <size>
             <width>100</width>
             <height>0</height>
            </size>
           </property>
           <property name="text">
            <string>Remove Layer</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="maya_render_project_label">
         <property name="text">
          <string>Maya Project</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <layout class="QHBoxLayout" name="maya_render_project_layout">
         <item>
          <widget class="QLineEdit" name="maya_render_project_line">
           <property name="placeholderText">
            <string>Maya Project Path</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="maya_render_project_btn">
           <property name="text">
            <string>Browse</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="maya_render_output_label">
         <property name="text">
          <string>Output</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <layout class="QHBoxLayout" name="maya_render_output_grp">
         <item>
          <widget class="QLineEdit" name="maya_render_output_line">
           <property name="placeholderText">
            <string>Default</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="maya_render_output_btn">
           <property name="text">
            <string>Browse</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="mayapy_grp">
      <property name="title">
       <string>MayaPy</string>
      </property>
      <property name="flat">
       <bool>true</bool>
      </property>
      <layout class="QFormLayout" name="formLayout_7">
       <item row="0" column="0">
        <widget class="QLabel" name="mayapy_script_label">
         <property name="text">
          <string>Script</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QPlainTextEdit" name="mayapy_script_text">
         <property name="sizeAdjustPolicy">
          <enum>QAbstractScrollArea::AdjustToContents</enum>
         </property>
         <property name="placeholderText">
          <string>Python Script</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="command_grp">
      <property name="title">
       <string>Command</string>
      </property>
      <property name="flat">
       <bool>true</bool>
      </property>
      <layout class="QFormLayout" name="formLayout_8">
       <item row="0" column="0">
        <widget class="QLabel" name="command_label">
         <property name="text">
          <string>Command</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="command_line">
         <property name="placeholderText">
          <string>Arbitrary Command</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="advanced_grp">
      <property name="title">
       <string>Advanced</string>
      </property>
      <property name="flat">
       <bool>true</bool>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <layout class="QFormLayout" name="advanced_layout">
         <item row="0" column="0">
          <widget class="QLabel" name="args_label">
           <property name="text">
            <string>Args</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLineEdit" name="args_line">
           <property name="placeholderText">
            <string>Extra Args</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="staus_label">
           <property name="text">
            <string>Status</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <layout class="QHBoxLayout" name="status_layout">
           <item>
            <widget class="QRadioButton" name="status_ready_button">
             <property name="text">
              <string>Ready</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QRadioButton" name="status_paused_button">
             <property name="text">
              <string>Paused</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="advanced_spin_layout">
         <item>
          <layout class="QFormLayout" name="advanced_spin_layout_l">
           <item row="0" column="0">
            <widget class="QLabel" name="priority_label">
             <property name="text">
              <string>Priority</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QSpinBox" name="priority_spinbox">
             <property name="minimum">
              <number>1</number>
             </property>
             <property name="maximum">
              <number>100</number>
             </property>
             <property name="value">
              <number>50</number>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="max_nodes_label">
             <property name="text">
              <string>Max Nodes</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QSpinBox" name="max_nodes_spinbox">
             <property name="maximum">
              <number>99999</number>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QFormLayout" name="advanced_spin_layout_r">
           <item row="0" column="0">
            <widget class="QLabel" name="timeout_label">
             <property name="text">
              <string>Timeout</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QSpinBox" name="timeout_spinbox">
             <property name="maximum">
              <number>99999</number>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="max_failures_label">
             <property name="text">
              <string>Max Failures</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QSpinBox" name="max_failures_spinbox">
             <property name="maximum">
              <number>99999</number>
             </property>
             <property name="value">
              <number>6</number>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="submit_btn">
      <property name="text">
       <string>Submit</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
